apps/web/components/AppListCard.tsx

```diff
@@ -19,6 +20,7 @@ type AppListCardProps = {
   isTemplate?: boolean;
   invalidCredential?: boolean;
   children?: ReactNode;
+  credentialOwner?: CredentialOwner;
```

Comment:
I think we should make it a required field and expect `null` to be passed explicitly if someone doesn't want to pass `credentialOwner`. It helps in catching unexpected misses in prop passing.

I see CalendarListContainer(and other places) not passing the prop? Is that intentional?
<img width="747" alt="Screenshot 2023-06-14 at 6 05 21 PM" src="https://github.com/calcom/cal.com/assets/1780212/506cb1b7-dae6-4242-857e-33d240367bd1">

Reply:
The places where we don't want to pass the `credentialOwner` prop are the apps where we don't want to be installed team-wide (calendars & videos for now).